test/functional/master/devportal/auth_controller_test.rb

```diff
@@ -5,14 +5,14 @@
 class Master::Devportal::AuthControllerTest < ActionController::TestCase
 
   setup do
-    host! master_account.domain
+    host! master_account.internal_domain
 
     @provider = FactoryBot.create(:simple_provider)
     @authentication_provider = FactoryBot.create(:authentication_provider, account: @provider)
   end
 
   test '#show must redirect to the login dev portal of the provider' do
-    get :show, params: { domain: @provider.domain, system_name: @authentication_provider.system_name, code: 'A1234', plan_id: 42 }
-    assert_redirected_to "http://#{@provider.domain}/auth/#{@authentication_provider.system_name}/callback?code=A1234&master=true&plan_id=42"
+    get :show, params: { domain: @provider.internal_domain, system_name: @authentication_provider.system_name, code: 'A1234', plan_id: 42 }
+    assert_redirected_to "http://#{@provider.external_domain}/auth/#{@authentication_provider.system_name}/callback?code=A1234&master=true&plan_id=42"
```

Comment:
Are these two different in this case?

Reply:
I followed what's described in https://github.com/3scale/porta/blob/bbd1a6c23cb66031a8c243e41c252d82fa2cd3eb/lib/three_scale/domain_substitution.rb#L14-L96

Reply:
But no they are not different.

Reply:
Cool. If we do changes, I guess, these will need to be re-reviewed..